### PR TITLE
Cover Test for Configuration Folder

### PR DESCRIPTION
**Pull request by jesusHCG**
Description
Code coverage for the configuration folder was increased

Related GitHub issue or JIRA story
https://jira.autodesk.com/browse/DYN-2485

Checklist
[  ] PR title reflects the changes it introduces. Consider using this template: QNTM-XXXX: Implement X Feature
[ - ] Code compiles correctly
[ - ] No linting errors
[ - ] All tests passing
[  ] Updated README.md / documentation (if necessary)

### CODE REVIEW

File: test/DynamoCoreTests/DynamoCoreTests.csproj [hunk 1]
```diff
@@ -237,6 +240,7 @@
   <ItemGroup>
     <None Include="packages.config" />
   </ItemGroup>
+  <ItemGroup />
```
Comment (mjkkirschner) [inline test/DynamoCoreTests/DynamoCoreTests.csproj]: can we remove this?

Comment (jesusHCG) [reply test/DynamoCoreTests/DynamoCoreTests.csproj]: I'm not sure, the file was already in the project

Comment (jesusHCG) [reply test/DynamoCoreTests/DynamoCoreTests.csproj]: I´m doing some test.

Comment (jesusHCG) [reply test/DynamoCoreTests/DynamoCoreTests.csproj]: yes, I already remove this line

**Comment by jesusHCG** [comment]
On my local environment it gives me the next result.

image

**Comment by QilongTang** [comment]
LGTM then

**Pull request merged**